reference/git_github.rst

```diff
@@ -3,6 +3,16 @@ Git and Github
 
 This document describes some best practices for using Git and Github.
 
+Best practises
+--------------
+- Create a separate pull request for each bug. If you have multiple commits in one pull request, squash them into one.
```

Comment:
This depends on how complicated the change is. Complex changes can be easier to review as a series of small, self-contained commits.